core/src/main/scala/io/delta/tables/DeltaTableBuilder.scala

```diff
@@ -17,17 +17,17 @@
 package io.delta.tables
 
 import scala.collection.mutable
-
```

Comment:
nit: we have blank lines separating imports from different groups (e.g. scala vs other vs spark)